src/templates/kit/screens/ReviewScreen.js

```diff
@@ -9,13 +9,13 @@ import {
   useKeyDown,
   useTheme,
 } from '@aragon/ui'
-import { Header, Navigation } from '..'
+import { Header, Navigation, ScreenPropsType } from '..'
 
-function Review({
-  screenProps: { back, data, next },
+function ReviewScreen({
   items,
-  screenTitle = 'Review information',
+  screenProps: { back, data, next },
   screenSubtitle = 'Have one last look at your settings below',
+  screenTitle = 'Review information',
```

Comment:
I think we can remove these defaults, since they're supplied by the default props now